### DISCUSSION TITLE
How to access results from a rstp-file using dpf?

### DISCUSSION THREAD

**Discussion by bchocholaty** [ansys/pydpf-core#1211] [Q&A]
Is there a way to access results from a rstp-file? The rstp-file results from a prestressed modal analysis which uses perturbation analysis.

For rst-files I do the following, which works nicely:

    model = dpf.Model(rst_file)
    disp = model.results.displacement.on_all_time_freqs.eval()
    
However, if I do the same for file.rstp, I get the following error:

ansys.dpf.gate.errors.DPFServerException: MeshProvider:5<-meshes_provider:6<-Operator(s) (mapdl::rstp::MeshProvider) does not exists in the registry !

**Comment by RobPasMue** [comment]
Transferring discussion to the DPF repository =)
